zio-http/src/main/scala/zio/http/model/Cookie.scala

```diff
@@ -1,8 +1,9 @@
-package zio.http
+package zio.http.model
 
-import zio.http.Cookie.{SameSite, Type}
 import zio.http.CookieDecoder.log
+import zio.http.model.Cookie.{SameSite, Type}
 import zio.http.service.Log
```

Comment:
At least this service import seems odd.

Reply:
Yeah I would not log an error on this level